remo/reports/migrations/0016_populate_country_field.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+from product_details import product_details
+
+COUNTRIES_LIST = product_details.get_regions('en').values()
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """ Populate the country field in the reports based on the location
+        submitted by the user or the location of the event linked to the
+        report.
+        """
+        Report = orm['reports.NGReport']
+        events_reports = Report.objects.filter(event__isnull=False)
+
+        for report in events_reports:
+            report.country = report.event.country
+            report.save()
+
+        events_reports_ids = events_reports.values_list('id', flat=True)
+        reports = Report.objects.exclude(id__in=events_reports_ids)
+
+        for report in reports:
+            try:
+                country = report.location.split(',')[-1].strip()
+            except IndexError:
+                country = ''
+            if country in COUNTRIES_LIST:
+                self.country = country
+            report.country = country
```

Comment:
By doing this we won't have a default value for countries. On the other hand I could change the model to provide a default value.